app/views/tag_groups/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<% modal ||= false %>
+<% remote = modal ? true : false %>
+
+<%= form_for(tag_group, remote: remote, html: {role: :form, 'data-model' => 'tag_group'}) do |f| %>
```

Comment:
You can avoid using strings for HTML attributes doing like this:

`{ role: :form, data: { model: 'tag_group' } }`